terraspace/app/stacks/peer-e2e-testing/variables.tf

```diff
@@ -0,0 +1,24 @@
+variable "subnet_id" {
+  type        = string
+  description = "ID of subnet where peer_e2e_testing should run"
+}
+
+variable "security_group_id" {
+  type        = string
+  description = "ID of security group where peer_e2e_testing should run"
+}
+
+variable "key_name" {
+  type        = string
+  description = "AWS key Pair name"
+}
+
+variable "ec2_instance_name" {
+  type        = string
+  description = "Name for the EC2 which will run bucket mirror"
```

Comment:
Aha! Found a "bucket mirror" wondering in wrong places :sweat_smile:

Reply:
Haha! Thanks, fixed!